### PR TITLE
Upcoming Events with react

### PR DESCRIPTION
**Pull request by MAlngaawy**
finish all but still
1_ date confusing
2_ can't set user from database
3_ not separated sections

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -307,94 +308,14 @@ class App extends React.Component {
...
-            </div>
+            <UpcomingEvents />
 
             <div className="w-3/5 relative z-20">
               <div className="absolute z-30">
-                <img
-                  src="./assets/img/events/bg_2.png"
-                  alt="upcoming events right-bg"
-                />
+                <img src="PNG Background" alt="upcoming events right-bg" />
```
Comment (Nilegfx) [inline src/App.js:318]: what is this?

File: src/components/UpcomingEvents/index.jsx [hunk 2]
```diff
@@ -0,0 +1,111 @@
...
+// Heading global
+import Heading from '../Heading'
+// Date Package
+import {parseISO} from 'date-fns';
+// Images
+import bg_1 from './img/bg_1.png';
+
+// Some Other Components
+export const BackgroundImage = () => (
+  <div className="upcoming-events__bg-image absolute w-2/5">
```
Comment (Nilegfx) [inline src/components/UpcomingEvents/index.jsx:13]: we can consider this in the future, I know you are not the original developer for this component, but I think it has some room for improvement (like having it as a background for the outer section or something, not sure)

File: src/components/UpcomingEvents/index.jsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+    <img src={bg_1} alt="upcoming events Background" />
+  </div>
+);
+
+const Events = props => {
+// Function to get add dates needed
+function getDate(myDate) {
```
Comment (Nilegfx) [inline src/components/UpcomingEvents/index.jsx:20]: I ADMIRE how you 'UNBLOCKED' yourself and applied the **RESULT-ORIENTED** concept perfectly in this function. 
For other team members, This is a good example/approach to follow when you don't know how to solve a problem, do not stick for a longer time, deliver the desired task and improve it iteratively (later)

however, this doesn't prevent me from suggesting the change 
```js
function getDate(myDate) {
    const theDate = parseISO(myDate);

    return{
      month: format(theDate, 'MMM'),
      day: format(theDate, 'dd'),
      year: format(theDate, 'yyyy'),
      time: format(theDate, 'hh:mm 	a').toLowerCase()
    };
  }
```
Very good job ya nag3awy!👍🏼

Comment (MAlngaawy) [reply src/components/UpcomingEvents/index.jsx:20]: Thanks I'm going to look to it

File: src/components/UpcomingEvents/index.jsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+import './styles.css';
+import { charityAPI } from '../../clients';
+// Heading global
+import Heading from '../Heading'
+// Date Package
+import {parseISO} from 'date-fns';
+// Images
+import bg_1 from './img/bg_1.png';
+
+// Some Other Components
```
[the remark this answers is not in the document]
Comment (MAlngaawy) [reply src/components/UpcomingEvents/index.jsx:11]: Ok

File: src/App.js (hunk 1 above)
Comment (MAlngaawy) [reply src/App.js:318]: This is the image in the red card .. the App.js doesn't import the src .. so i put any thing in the src

**Comment by elhaw** [comment]
I do not know the deployment has failed. Any suggestion @Nilegfx

**Pull request merged**